### PR TITLE
Add quick-start template for emptyDir volumes

### PR DESCRIPTION
**Pull request by zhedahht**
### Best Practice Checklist
Check these items before submitting a PR... See the Contribution Guide for the full detail: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md 

1. uri's compatible with all clouds (Stack, China, Government)
1. Staged artifacts use _artifactsLocation & _artifactsLocationSasToken
1. Use resourceGroup().location for resource locations
1. Folder names for artifacts (nestedtemplates, scripts, DSC)
1. Use literal values for apiVersion (no variables)
1. Parameter files (GEN-UNIQUE for value generation and no "changemeplease" values
1. $schema and other uris use https
1. Use uniqueString() whenever possible to generate names for resources.  While this is not required, it's one of the most common failure points in a deployment. 
1. Update the metadata.json with the current date

For details: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/bp-checklist.md

- [ ] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

### Changelog

*
*
*

### CODE REVIEW

File: 201-aci-linuxcontainer-volume-emptydir/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Azure Container Instances
+
+This template demonstrates a simple use case for emptyDir volume of [Azure Container Instances](https://docs.microsoft.com/en-us/azure/container-instances/).
```
Comment (bmoore-msft) [inline 201-aci-linuxcontainer-volume-emptydir/README.md]: Add the deploy to azure and visualize buttons (see other samples for example (non-aci ones))

File: 201-aci-linuxcontainer-volume-emptydir/azuredeploy.parameters.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{
```
Comment (bmoore-msft) [inline 201-aci-linuxcontainer-volume-emptydir/azuredeploy.parameters.json]: since you have defaultValues for these, just empty the parameters object...

Comment (zhedahht) [reply 201-aci-linuxcontainer-volume-emptydir/azuredeploy.parameters.json]: I tried to delete defaultValues of these parameters instead.

Comment (bmoore-msft) [reply 201-aci-linuxcontainer-volume-emptydir/azuredeploy.parameters.json]: No, we want defaultValues in the template whenever possible - leave the param file for GEN* values.  Makes it easier to deploy when the template has defaultValues

**Pull request merged**